src/ProjectsClient.ts

```diff
@@ -6,7 +6,7 @@
  * @module ProjectsClient
  */
 import type { AccessToken } from "@itwin/core-bentley";
-import { request, RequestOptions } from "@bentley/itwin-client";
+import axios, { AxiosRequestConfig } from "axios";
```

Comment:
Why is axios outside the import braces? I'm not familiar with this format

Reply:
axios is the default import, named imports are inside the braces

Reply:
Oh okay, thanks